remapping_pipeline.nf

```diff
@@ -0,0 +1,350 @@
+#!/usr/bin/env nextflow
+
+def helpMessage() {
+    log.info"""
+    Index the provided genome with bowtie and samtools
+    Usage:
+            --oldgenome                     Fasta file containing the reference genome.
+            --outdir                    Directory where the index will be deployed.
+    """
+}
+params.flankingseq = 50
+params.scorecutoff = 0.6
+params.diffcutoff =  0.04
+params.outfile = "$baseDir/resources/remap_vcf.vcf"
+
+// Show help message
+if (params.help) exit 0, helpMessage()
+params.oldgenome = "$baseDir/resources/genome.fa"
+params.newgenome = "$baseDir/resources/genome.fa"
+params.vcffile = "$baseDir/resources/source_vcf.vcf"
+
+// Index files for both old and new genomes 
+oldgenome_chrom_sizes = file("${params.oldgenome}.chrom.sizes")
+oldgenome_fai = file("${params.oldgenome}.fai")
+newgenome_dir = file(params.newgenome).getParent()
+newgenome_basename = file(params.newgenome).getName()
+newgenome_fai = file("${params.newgenome}.fai")
+
+// basename and basedir of the output file to know how to name the output file
+outfile_basename = file(params.outfile).getName()
+outfile_dir = file(params.outfile).getParent()
+
+
+/*
+ * Store the original VCF header for later use
+ */
+process StoreVCFHeader {
+
+    input:
+        path "source_vcf" from params.vcffile
+
+    output:
+        path "vcf_header.txt" into vcf_header
+
+    """
+    bcftools view --header-only source_vcf -o vcf_header.txt
+    """
+}
+
+/*
+ * Convert vcf file to bed format
+ */
+process ConvertVCFToBed {
+
+    input:
+        path "source_vcf" from params.vcffile
+
+    output:
+        path "variants.bed" into variants_bed
+
+    """
+    vcf2bed < source_vcf > variants.bed
+    """
+}
+
+/*
+ * Convert get the flanking region in bed format.
```

Comment:
```suggestion
 * Based on variants BED, generate the flanking regions BED.
```

Reply:
@tcezard This has been resolved but not applied — is this accidental, or you don't agree with the new wording? :)